python/paddle/nn/initializer/uniform.py

```diff
@@ -158,24 +158,24 @@ class Uniform(UniformInitializer):
     Examples:
         .. code-block:: python
 
-            import paddle
-
-            data = paddle.ones(shape=[3, 1, 2], dtype='float32')
-            weight_attr = paddle.framework.ParamAttr(
-                name="linear_weight",
-                initializer=paddle.nn.initializer.Uniform(low=-0.5, high=0.5))
-            bias_attr = paddle.framework.ParamAttr(
-                name="linear_bias",
-                initializer=paddle.nn.initializer.Uniform(low=-0.5, high=0.5))
-            linear = paddle.nn.Linear(2, 2, weight_attr=weight_attr, bias_attr=bias_attr)
-            # linear.weight:  [[-0.46245047  0.05260676]
-            #                  [ 0.38054508  0.29169726]]
-            # linear.bias:  [-0.2734719   0.23939109]
-
-            res = linear(data)
-            # res:  [[[-0.3553773  0.5836951]]
-            #        [[-0.3553773  0.5836951]]
-            #        [[-0.3553773  0.5836951]]]
+            >>> import paddle
+
+            >>> data = paddle.ones(shape=[3, 1, 2], dtype='float32')
+            >>> weight_attr = paddle.framework.ParamAttr(
+            ...     name="linear_weight",
+            ...     initializer=paddle.nn.initializer.Uniform(low=-0.5, high=0.5))
+            >>> bias_attr = paddle.framework.ParamAttr(
+            ...     name="linear_bias",
+            ...     initializer=paddle.nn.initializer.Uniform(low=-0.5, high=0.5))
+            >>> linear = paddle.nn.Linear(2, 2, weight_attr=weight_attr, bias_attr=bias_attr)
+            >>> # linear.weight:  [[-0.46245047  0.05260676]
+            >>> #                  [ 0.38054508  0.29169726]]
+            >>> # linear.bias:  [-0.2734719   0.23939109]
+
+            >>> res = linear(data)
+            >>> # res:  [[[-0.3553773  0.5836951]]
+            >>> #        [[-0.3553773  0.5836951]]
+            >>> #        [[-0.3553773  0.5836951]]]
```

Comment:
同理，如无特殊情况，请勿使用 # 的注释方式提供输出值

- 另外，`paddle.nn.initializer.Uniform`有随机性，请在代码开始出加`seed`

```suggestion
            >>> print(linear.weight)
            Parameter containing:
            Tensor(shape=[2, 2], dtype=float32, place=Place(cpu), stop_gradient=False,
                   [[ 0.36583614,  0.02014720],
                    [-0.24039063,  0.40525323]])
                   
            >>> print(linear.bias)
            Parameter containing:
            Tensor(shape=[2], dtype=float32, place=Place(cpu), stop_gradient=False,
                   [-0.07599911, -0.09358713])
                   
            >>> res = linear(data)
            >>> print(res)  
            Tensor(shape=[3, 1, 2], dtype=float32, place=Place(cpu), stop_gradient=False,
                   [[[0.04944640, 0.33181331]],

                    [[0.04944640, 0.33181331]],

                    [[0.04944640, 0.33181331]]])
```